src/engine/Eval_generic.mli

```diff
@@ -5,6 +5,7 @@ type value =
   (* the string does not contain the enclosing '"' *)
   | String of string
   | List of value list
+  | Date of int * int * int
```

Comment:
Could you add a comment here in the `.mli` too, about the meaning of each of the `int`s?

Reply:
Done!